client/package.json

```diff
@@ -19,7 +19,10 @@
     ],
     "babel": {
         "presets": [
-            "env"
+            "env",
+            "es2015",
+            "stage-0",
+            "react"
```

Comment:
I think we could pull this back out too, the preact-cli has a preact-babel preset it uses don't want to risk those colliding if we don't have to.